src/tag/index.ts

```diff
@@ -0,0 +1,60 @@
+import {AxiosInstance} from 'axios'
+import {Tag, TagDescription} from './tag'
+import {Entry} from '../entry/entry'
+import {Journal} from '../journal/journal'
+
+export const createTags = async (
+    spireClient: AxiosInstance, 
+    journalId: Journal['id'],
+    entryId: Entry['id'],
+    tag: Tag['name']
+): Promise<string[]> =>{
```

Comment:
change to `Promise<Response<Tag[]>>` change everywhere

- [ ] do not return array of primitive types, always objects
- [ ] all function should return same response object